### PR TITLE
Add Single application for CloudCompare in OoD

### PR DESCRIPTION
**Pull request by ktiits**

### CODE REVIEW

**Review by samumantha — COMMENTED** [review]
2 comments added

File: docs/apps/cloudcompare.md [hunk 1]
```diff
@@ -14,18 +14,24 @@ The following versions of CloudCompare are available in **Puhti**:
...
+1. Log in to [Puhti web interface](https://puhti.csc.fi). 
+2. Start CloudCompare: Apps -> Desktop, choose Desktop: 'single application' and App: 'CloudCompare'
+3. CloudCompare is started automatically when the Desktop is launched. 
 
-```bash
 
+Alternatively, especially if you want to use CloudCompare together with some other Graphical User Interface (GUI) tool or want to use data from Allas, CloudCompare can be started in Puhti web interface with remote desktop:
+
+1. Log in to [Puhti web interface](https://puhti.csc.fi).
+2. Open Remote desktop: Apps -> Desktop, choose Desktop: `mate` or `xfce`.
```
[the remark this answers is not in the document]
Comment (samumantha) [reply docs/apps/cloudcompare.md:27]: Do you mean "Desktop"?

[the remark this answers is not in the document]
Comment (samumantha) [reply docs/apps/cloudcompare.md:27]: image

[the remark this answers is not in the document]
Comment (ktiits) [reply docs/apps/cloudcompare.md:27]: It is under: Apps -> Desktop
It is exactly the same text for all 5 GIS GUI tools..

File: docs/apps/cloudcompare.md [hunk 2]
```diff
@@ -14,18 +14,24 @@ The following versions of CloudCompare are available in **Puhti**:
...
+2. Start CloudCompare: Apps -> Desktop, choose Desktop: 'single application' and App: 'CloudCompare'
+3. CloudCompare is started automatically when the Desktop is launched. 
 
-```bash
 
+Alternatively, especially if you want to use CloudCompare together with some other Graphical User Interface (GUI) tool or want to use data from Allas, CloudCompare can be started in Puhti web interface with remote desktop:
+
+1. Log in to [Puhti web interface](https://puhti.csc.fi).
+2. Open Remote desktop: Apps -> Desktop, choose Desktop: `mate` or `xfce`. 
+3. After launcing the remote desktop open `Host Terminal` (Desktop icon) and start CloudCompare:
```
[the remark this answers is not in the document]
Comment (ktiits) [reply docs/apps/cloudcompare.md:28]: Will add the icon option to all 5 tomorrow, I want to keep the commands also still visible.

File: docs/apps/cloudcompare.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samumantha) [reply docs/apps/cloudcompare.md:27]: Hmm, I think "remote desktop" still might be confusing for users as a term here?
But maybe also just for my afternoon brain

File: docs/apps/cloudcompare.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ktiits) [reply docs/apps/cloudcompare.md]: Added icon usage

**Review by attesillanpaa — APPROVED** [review]

**Pull request merged**